pkg/controller/chaosscheduler/utils.go

```diff
@@ -0,0 +1,218 @@
+package chaosscheduler
```

Comment:
change this file name

Reply:
As this file contains general miscellaneous functions.. I think this is the best suitable file name.

Reply:
PTAL: https://github.com/litmuschaos/litmus/blob/cfd91c26ca1ca668a40d2bc85fdbc23f576de8df/contribute/developer_guide/STYLE_GUIDE.md#package-naming

Reply:
Changed